configure.py

```diff
@@ -487,8 +496,8 @@ def process_command_line(args): # pylint: disable=too-many-locals,too-many-state
     build_group.add_option('--with-debug-asserts', action='store_true', default=False,
                            help=optparse.SUPPRESS_HELP)
 
-    build_group.add_option('--build-bogo-shim', action='store_true', default=False,
-                           help=optparse.SUPPRESS_HELP)
+    build_group.add_option('--build-targets', default=None, dest="build_targets",
+                           help="build specific targets and tools (%s)" % ', '.join(ACCEPTABLE_BUILD_TARGETS))
```

Comment:
If you use an `append` target then it becomes possible to provide multiple --build-target options on the command line. That is sometimes useful when the configure.py invocation is being created using a script (as is done, in various ways, in most distribution packages). Ex some distribution script might have logic like

```
if build_cli:
   cmd += ['--build-target=cli']
if build_shared:
   cmd += ['--build-target=shared']
...
```

Depending on the language used it may be easier to provide multiple options vs performing a join to form a single --build-targets=cli,shared.

Reply:
Both is possible now. Even something like `--build-targets=cli --build-targets="static,shared"`, is flattened accordingly.